src/style.css

```diff
@@ -0,0 +1,146 @@
+/* #region Normalize */
+
+:root {
+  --main-accent: #060b35;
+  --secondary: #616070;
+  --white: #fff;
+  --blue-accent: #00acdc;
+  --gray-elements: #e5e5e5;
+  --yellow-stars: #ffde6a;
+  --card-border-color: #f3f3f3;
+  --card-border: 1px solid var(--card-border-color);
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  background-color: var(--white);
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  color: var(--main-accent);
+}
+
+p {
+  margin: 0;
+}
+
+/* #endregion */
+
+.card {
+  width: 166px;
```

Comment:
card should be 200px
apply `box-sizing: border-box;` to `*` selector and define 200px here